readme.md

```diff
@@ -32,7 +32,7 @@ Returns an array of matching paths.
 
 Type: `string | string[]`
 
-Paths to match against.
+Paths to match against in the order of input paths.
```

Comment:
Not the correct spot

Reply:
No other place fits properly for this info. Can you please suggest a place to add it? or shall I create a new section to add this info?

Reply:
I think it's pretty clear if you look at it closer. This is the `path` input variable docs. Your addition is about the return value.

Reply:
fixed